### DISCUSSION TITLE
隐藏/i/目录选项不可用

### DISCUSSION THREAD

**Discussion by baddate** [icret/EasyImages2.0#142] [问答] [answered]
![image](https://github.com/icret/EasyImages2.0/assets/37013819/1844c5f0-dcc8-46e2-9c48-1ecd0b0c6715)

这种是什么原因？

p.s. 我的系统配置，用的是共享主机

![image](https://github.com/icret/EasyImages2.0/assets/37013819/45ace340-f015-4e49-b501-3643a1c8f68b)

**Comment by icret** [comment, marked as answer]
鼠标放到上边会有提示你怎么设置
